test/add-retrieve.spec.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-env mocha */
```

Comment:
This was originally intended as a set of reusable tests to run after the different combinations of `spawn/init/start` to make sure the daemon were being created in a functional way - this intent is lost with this change. How are we testing for this cases?